igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/client/JenkinsClient.groovy

```diff
@@ -40,7 +40,11 @@ interface JenkinsClient {
     @GET('/api/xml?tree=jobs[name,lastBuild[actions[failCount,skipCount,totalCount,urlName],duration,number,timestamp,result,building,url]]&exclude=/*/*/*/action[not(totalCount)]')
     ProjectsList getProjects()
 
-    @GET('/api/xml?tree=jobs[name]')
+    /*
+     * Jobs created with the Jenkins Folders plugin are nested.
+     * This query looks for jobs within folders with a depth of 10.
+     */
+    @GET('/api/xml?tree=jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name]]]]]]]]]]')
```

Comment:
in the ideal world this kind of structure and resolution would be configurable at the jenkins masters level.